keps/sig-api-machinery/20190711-remove-selflink.md

```diff
@@ -134,8 +134,6 @@ I went through a k/k repo (including its staging repos) and all repos under [kub
 and this is the list of places that reference `SelfLink` fields (I excluded tests and all places
 in apiserver responsible for setting it):
 
-- [ ] https://github.com/kubernetes/kubernetes/blob/master/pkg/api/ref/ref.go
```

Comment:
There were two copies of that file in the codebase - this one and staging/src/k8s.io/client-go/tools/reference/ref.go

I migrated stuff to the latter and removed this one.

Reply:
I wouldn't removed it from the KEP though - it was there when the KEP was created.

Reply:
ok,this is fine.